packages/xrpl/test/integration/transactions/ammCreate.test.ts

```diff
@@ -0,0 +1,45 @@
+import { assert } from 'chai'
+import { isValidClassicAddress } from 'xrpl'
+
+import { AMMInfoResponse } from '../../../src'
+import serverUrl from '../serverUrl'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
+import { createAMMPool } from '../utils'
+
+describe('AMMCreate', function () {
+  let testContext: XrplIntegrationTestContext
+
+  beforeAll(async () => {
+    testContext = await setupClient(serverUrl)
+  })
+  afterAll(async () => teardownClient(testContext))
+
+  it('base', async function () {
```

Comment:
Could you add more of what it "should" do? Across all the files

Reply:
That's what the model docs are for.

Reply:
When the test runs it will show `ammCreate base` which doesn't seem useful when you're trying to follow it. It's a nit though, so up to omar

Reply:
That's what all the basic tests show for the models.

Reply:
I'd like to keep it consistent with all test models. I think base is self-explanatory.